src/bandersnatch/mirror.py

```diff
@@ -970,6 +970,7 @@ async def mirror(
         )
 
     mirror_url = config.get("mirror", "master")
+    allow_non_https = config.getboolean("mirror", "allow-non-https", fallback=None)
```

Comment:
```suggestion
    allow_non_https = config.getboolean("mirror", "allow-non-https", fallback=False)
```
None is falsey, but shouldn't we try keep the type the same since it's possible here?